tests/CommentEndpointTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+use Suyabay\Episode;
+use Suyabay\Comment;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class CommentEndpointTest extends TestCase
+{
+	/**
+     * Test that episode does not exist when a user try to
+     * access a route passing unexisting episode name.
+     *
+     * @return void
+     */
+    public function testThatEpisodeDoesNotExist()
+    {
+        $user = factory('Suyabay\User')->create();
+
+        $response = $this->actingAs($user)->call('GET', '/api/v1/episodes/ogabos/comments');
+        $decodeResponse = json_decode($response->getContent());
+        
+        $this->assertEquals($decodeResponse->message, 'Episode does not exist');
+        $this->assertEquals($response->status(), 404);   
+    }
+
+    /**
+     * Test that their are no comment available in an
+     * episode a user tries to access.
+     *
+     * @return void
+     */
+    public function testThatCommentIsNotAvailableForEpisode()
+    {
+        $user = factory('Suyabay\User')->create();
+
+        $this->createEpisode();
+
+        $response       = $this->actingAs($user)->call('GET', '/api/v1/episodes/ogaboss/comments');
+        $decodeResponse = json_decode($response->getContent());
+        $this->assertEquals($decodeResponse->message, 'Comment not available for this episode');
+        $this->assertEquals($response->status(), 404);
+    }
+
+    /**
+     * Test that a user gets all the comments available 
+     * in an episode.
+     *
+     * @return void
+     */
+    public function testgetEpisodeComments()
+    {
+        $user = factory('Suyabay\User')->create();
+
+        $this->createEpisode();
+        $this->createComment();
+
+        $response       = $this->actingAs($user)->call('GET', '/api/v1/episodes/ogaboss/comments');
+        $decodeResponse = json_decode($response->getContent());
+        $array          = $decodeResponse->data;
+        $this->assertTrue(is_array($decodeResponse->data));
+
+        $this->get('/api/v1/episodes/ogaboss/comments')
+            ->seeJson()
+            ->seeStatusCode(200);
+    }
+
+    /**
+     * Test that a user gets all the comments available 
+     * if query is passed.
+     *
+     * @return void
+     */
+    public function testdisplayCommentsByDate()
+    {
+        $user = factory('Suyabay\User')->create();
+
+        $this->createEpisode();
+        $this->createComment();
+
+        $response = $this->actingAs($user)->call('GET', '/api/v1/episodes/ogaboss/comments?fromDate=2016-05-10&toDate=2016-05-18&limit=2');
+        $decodeResponse = json_decode($response->getContent());
+        $array = $decodeResponse->data;
+        $this->assertTrue(is_array($decodeResponse->data));
```

Comment:
**Code Quality - Nitpick**
@andela-araimi Good Testing, but there is a possibility that the array is empty or the array count is less or more. So Please assert for that also.